b2

```diff
@@ -25,7 +25,7 @@ import getpass
 import hashlib
 import httplib
 import json
-import os.path
+import os.path, os.environ
```

Comment:
Please keep the imports ordered alphabetically.

Also that import fails in our CI environment. Which version of python have you tested it on?